addons/common/fnc_isTurnedOut.sqf

```diff
@@ -27,54 +27,48 @@ Examples:
 
 Author:
     Nou, courtesy of ACRE project.
+    shukari
 ---------------------------------------------------------------------------- */
 SCRIPT(isTurnedOut);
 
 params ["_unit"];
 
-private _turn = false;
-private _out = false;
+if (isNull objectParent _unit) exitWith {true};
+if (isTurnedOut _unit) exitWith {true};
+
+private _return = false;
 private _vehicle = vehicle _unit;
 private _cfg = configFile >> "CfgVehicles" >> (typeOf _vehicle);
 
-if (_vehicle != _unit) then {
-    if (driver _vehicle == _unit) then {
-        private _forceHideDriver = getNumber(_cfg >> "forceHideDriver");
-        _turn = (_forceHideDriver != 1);
-        _out = (_vehicle animationPhase "hatchDriver") > 0;
-    } else {
-        private _assignedRole = assignedVehicleRole _unit;
-        if (_assignedRole select 0 == "Turret") then {
-            private _turretPath = _assignedRole select 1;
-            private _turret = [_vehicle, _turretPath] call CBA_fnc_getTurret;
-
-            private _canHideGunner = getNumber(_turret >> "canHideGunner");
-            private _forceHideGunner = getNumber(_turret >> "forceHideGunner");
-            _turn = (_canHideGunner == 1 && _forceHideGunner == 0);
-
-            private _hatchAnimation = getText(_turret >> "animationSourceHatch");
-            _out = (_vehicle animationPhase _hatchAnimation) > 0;
+if (driver _vehicle == _unit) then {
+    _return = (_vehicle animationPhase "hatchDriver") > 0;
+} else {
+    private _assignedRole = assignedVehicleRole _unit;
```

Comment:
This command reports `nil` while remote controlling a unit. Try this function while remote controlling an AI with zeus, and it should fail.